lib/customErrorHandler.js

```diff
@@ -3,14 +3,19 @@ const debug = require('debug')('swagger-endpoint-validator:custom-error-handler'
 /**
  * Adds a custom error handler for the errors thrown by the validator.
  * @param {Object} app - Express application object.
+ * @param {Object} options - Configuration options.
```

Comment:
Again, I'm not sure if adding here the only field expected in this object is better for readibility or not.

Reply:
I'm following the same convention as in `lib/normalizeOptions.js`. It's true that there more options are used. However, if we plan to keep adding new options, it could become a bit difficult to maintain. Anyway, all the options are documented in `index.js`